kitsune/sumo/static/js/gallery.js

```diff
@@ -203,11 +208,15 @@
             var file = $input[0].files[0],
                 type = $input.attr('name');
             var file_ext = file.name.split(/\./).pop().toLowerCase();
-            return (in_array(file_ext, CONSTANTS.extensions[type]) &&
-                    file.size < CONSTANTS.max_size[type]);
+            return (in_array(file_ext, CONSTANTS.extensions[type]));
+        },
+        isNotTooLarge: function ($input) {
+            var file = $input[0].files[0],
+                type = $input.attr('name');
+            return (file.size < CONSTANTS.max_size[type]);
         },
         /*
-         * Fired when upload starts, if isValidFile returns true
+         * Fired when upload starts, if isValidFile and isNotTooLarge return true
```

Comment:
This comment shouldn't reference `isNotTooLarge`, because that function is entirely a part of `isValidFile`. Sine `startUpload` doesn't make any direct reference to `isNotTooLarge`, this is kind of confusing.